students/jeongyong/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from .models import User
+
+
+class SignUpView(View):
+    def post(self, request):
+        try :    
+            data     = json.loads(request.body)
+            name     = data['name']
+            email    = data['email']
+            contact  = data['contact']
+            password = data['password']
+       
+            if not re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$' , email) :
+                return JsonResponse({'messasge':'이메일 형식에 맞지 않습니다.'}, status=400)
```

Comment:
응답 메시지는 최대한 영어로 작성해 주세요.
다른 부분들도 마찬가지 입니다.

```suggestion
            if not re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$' , email) :
                return JsonResponse({'messasge':'Invalid Email'}, status=400)
```